WordPressUI/Extensions/UIImageView+Networking.swift

```diff
@@ -128,7 +128,7 @@ public extension UIImageView {
 
     /// Stores the Image's remote URL, if any.
     ///
-    private var downloadURL: URL? {
+    internal var downloadURL: URL? {
```

Comment:
By default visibility is `internal`, no need to specify it.

Reply:
The extension is `public extension UIImageView` so the default access level for this extension is `public`.

Reply:
TIL that's only valid for extensions. Thanks Nate!!!